src/internal/packager2/find_images.go

```diff
@@ -482,6 +490,30 @@ func appendToImageMap(imgMap map[string]bool, pod corev1.PodSpec) map[string]boo
 	return imgMap
 }
 
+func appendToImageMapOCIRepo(imgMap map[string]bool, repo sourcev1beta2.OCIRepositorySpec) map[string]bool {
+	var regex = regexp.MustCompile(`oci://(.+)`)
+	var t = regex.FindStringSubmatch(repo.URL)
+
+	s := strings.Builder{}
+	// Pulls out just the url from the regex
+	s.Write([]byte(t[1]))
+
+	if repo.Reference.Tag != "" {
+		s.WriteString(":")
+		s.WriteString(repo.Reference.Tag)
+	} else if repo.Reference.Digest != "" {
+		s.WriteString("@")
+		s.WriteString(repo.Reference.Digest)
+	} else if repo.Reference.SemVer != "" || repo.Reference.SemverFilter != "" {
+		fmt.Errorf("Can not use semver or semverFilter with OCIRepository")
```

Comment:
```suggestion
		logger.From(ctx).Error("can not use semver or semverFilter with OCIRepository")
```
Pass context here and use the builtin logger

Reply:
Updated per comment